internal/errors/errors.go

```diff
@@ -0,0 +1,162 @@
+package errors
```

Comment:
A package as widely used as this would certainly benefit from some unit tests

Reply:
:100:

Reply:
at the same time, the functions are so simple that i'm not really sure how much tests would add here

Reply:
i guess i could at least test interop with pkg/errors and sentry